sources/constraint-metadata.asciidoc

```diff
@@ -386,7 +386,7 @@ include::{validation-api-source-dir}javax/validation/metadata/ReturnValueDescrip
 [[constraintmetadata-containerdescriptor]]
 === ContainerDescriptor and ContainerElementTypeDescriptor
 
-The `ContainerDescriptor` interface describes those elements that can host container element constraints (see <<constraintdeclarationvalidationprocess-containerelementconstraints>>).
+The `ContainerDescriptor` interface describes those elements that can be of a container type such `List` or `Map` and as such may host container element constraints (see <<constraintdeclarationvalidationprocess-containerelementconstraints>>) or have container element types that are marked with `@Valid`.
```

Comment:
I think you're missing a `as` after the first `such`. But it's quite weird to have `as such` just after `such as`. Maybe replacing the first `such` with `e.g.` would be in order?